[2025-10-07T10:16:42.101+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-10-07T10:16:42.256+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: vnstock_weekly_etl_incremental.process_batch manual__2025-10-07T10:01:57.519057+00:00 map_index=12 [queued]>
[2025-10-07T10:16:42.310+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: vnstock_weekly_etl_incremental.process_batch manual__2025-10-07T10:01:57.519057+00:00 map_index=12 [queued]>
[2025-10-07T10:16:42.315+0000] {taskinstance.py:2867} INFO - Starting attempt 1 of 3
[2025-10-07T10:16:42.426+0000] {taskinstance.py:2890} INFO - Executing <Mapped(_PythonDecoratedOperator): process_batch> on 2025-10-07 10:01:57.519057+00:00
[2025-10-07T10:16:42.640+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=297) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-10-07T10:16:42.680+0000] {standard_task_runner.py:72} INFO - Started process 475 to run task
[2025-10-07T10:16:42.716+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'vnstock_weekly_etl_incremental', 'process_batch', 'manual__2025-10-07T10:01:57.519057+00:00', '--job-id', '15', '--raw', '--subdir', 'DAGS_FOLDER/dag.py', '--cfg-path', '/tmp/tmpaixz28_4', '--map-index', '12']
[2025-10-07T10:16:42.719+0000] {standard_task_runner.py:105} INFO - Job 15: Subtask process_batch
[2025-10-07T10:16:42.851+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/settings.py:209 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2025-10-07T10:16:43.309+0000] {task_command.py:467} INFO - Running <TaskInstance: vnstock_weekly_etl_incremental.process_batch manual__2025-10-07T10:01:57.519057+00:00 map_index=12 [running]> on host e3278e0f0bbc
[2025-10-07T10:16:50.642+0000] {taskinstance.py:3134} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='vnstock_weekly_etl_incremental' AIRFLOW_CTX_TASK_ID='process_batch' AIRFLOW_CTX_EXECUTION_DATE='2025-10-07T10:01:57.519057+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-10-07T10:01:57.519057+00:00'
[2025-10-07T10:16:50.645+0000] {taskinstance.py:732} INFO - ::endgroup::
[2025-10-07T10:16:50.670+0000] {logging_mixin.py:190} INFO - ⚠️ Lỗi khi xử lý PVT: 'Limiter' object has no attribute 'ratelimit'
[2025-10-07T10:16:50.675+0000] {logging_mixin.py:190} INFO - ⚠️ Lỗi khi xử lý PVS: 'Limiter' object has no attribute 'ratelimit'
[2025-10-07T10:16:50.685+0000] {logging_mixin.py:190} INFO - ⚠️ Lỗi khi xử lý PVR: 'Limiter' object has no attribute 'ratelimit'
[2025-10-07T10:16:50.686+0000] {logging_mixin.py:190} INFO - ⚠️ Lỗi khi xử lý PVP: 'Limiter' object has no attribute 'ratelimit'
[2025-10-07T10:16:50.687+0000] {logging_mixin.py:190} INFO - ⚠️ Lỗi khi xử lý PVO: 'Limiter' object has no attribute 'ratelimit'
[2025-10-07T10:16:50.688+0000] {logging_mixin.py:190} INFO - ⚠️ Lỗi khi xử lý PVM: 'Limiter' object has no attribute 'ratelimit'
[2025-10-07T10:16:50.690+0000] {logging_mixin.py:190} INFO - ⚠️ Lỗi khi xử lý PVL: 'Limiter' object has no attribute 'ratelimit'
[2025-10-07T10:16:50.700+0000] {logging_mixin.py:190} INFO - ⚠️ Lỗi khi xử lý PVI: 'Limiter' object has no attribute 'ratelimit'
[2025-10-07T10:16:50.702+0000] {logging_mixin.py:190} INFO - ⚠️ Lỗi khi xử lý PVH: 'Limiter' object has no attribute 'ratelimit'
[2025-10-07T10:16:50.703+0000] {logging_mixin.py:190} INFO - ⚠️ Lỗi khi xử lý PVG: 'Limiter' object has no attribute 'ratelimit'
[2025-10-07T10:16:50.706+0000] {logging_mixin.py:190} INFO - ⚠️ Lỗi khi xử lý PVE: 'Limiter' object has no attribute 'ratelimit'
[2025-10-07T10:16:50.715+0000] {logging_mixin.py:190} INFO - ⚠️ Lỗi khi xử lý PVD: 'Limiter' object has no attribute 'ratelimit'
[2025-10-07T10:16:50.718+0000] {logging_mixin.py:190} INFO - ⚠️ Lỗi khi xử lý PVC: 'Limiter' object has no attribute 'ratelimit'
[2025-10-07T10:16:50.718+0000] {logging_mixin.py:190} INFO - ⚠️ Lỗi khi xử lý PVB: 'Limiter' object has no attribute 'ratelimit'
[2025-10-07T10:16:50.719+0000] {logging_mixin.py:190} INFO - ⚠️ Lỗi khi xử lý PVA: 'Limiter' object has no attribute 'ratelimit'
[2025-10-07T10:16:50.719+0000] {logging_mixin.py:190} INFO - ⚠️ Lỗi khi xử lý PV2: 'Limiter' object has no attribute 'ratelimit'
[2025-10-07T10:16:50.720+0000] {logging_mixin.py:190} INFO - ⚠️ Lỗi khi xử lý PTX: 'Limiter' object has no attribute 'ratelimit'
[2025-10-07T10:16:50.721+0000] {logging_mixin.py:190} INFO - ⚠️ Lỗi khi xử lý PTV: 'Limiter' object has no attribute 'ratelimit'
[2025-10-07T10:16:50.732+0000] {logging_mixin.py:190} INFO - ⚠️ Lỗi khi xử lý PTT: 'Limiter' object has no attribute 'ratelimit'
[2025-10-07T10:16:50.741+0000] {logging_mixin.py:190} INFO - ⚠️ Lỗi khi xử lý PTS: 'Limiter' object has no attribute 'ratelimit'
[2025-10-07T10:16:50.743+0000] {logging_mixin.py:190} INFO - ⚠️ Lỗi khi xử lý PTP: 'Limiter' object has no attribute 'ratelimit'
[2025-10-07T10:16:50.748+0000] {logging_mixin.py:190} INFO - ⚠️ Lỗi khi xử lý PTO: 'Limiter' object has no attribute 'ratelimit'
[2025-10-07T10:16:50.753+0000] {logging_mixin.py:190} INFO - ⚠️ Lỗi khi xử lý PTN: 'Limiter' object has no attribute 'ratelimit'
[2025-10-07T10:16:50.754+0000] {logging_mixin.py:190} INFO - ⚠️ Lỗi khi xử lý PTM: 'Limiter' object has no attribute 'ratelimit'
[2025-10-07T10:16:50.757+0000] {logging_mixin.py:190} INFO - ⚠️ Lỗi khi xử lý PTL: 'Limiter' object has no attribute 'ratelimit'
[2025-10-07T10:16:50.761+0000] {logging_mixin.py:190} INFO - ⚠️ Lỗi khi xử lý PTI: 'Limiter' object has no attribute 'ratelimit'
[2025-10-07T10:16:50.765+0000] {logging_mixin.py:190} INFO - ⚠️ Lỗi khi xử lý PTH: 'Limiter' object has no attribute 'ratelimit'
[2025-10-07T10:16:50.768+0000] {logging_mixin.py:190} INFO - ⚠️ Lỗi khi xử lý PTG: 'Limiter' object has no attribute 'ratelimit'
[2025-10-07T10:16:50.775+0000] {logging_mixin.py:190} INFO - ⚠️ Lỗi khi xử lý PTE: 'Limiter' object has no attribute 'ratelimit'
[2025-10-07T10:16:50.777+0000] {logging_mixin.py:190} INFO - ⚠️ Lỗi khi xử lý PTD: 'Limiter' object has no attribute 'ratelimit'
[2025-10-07T10:16:50.782+0000] {logging_mixin.py:190} INFO - ⚠️ Lỗi khi xử lý PTC: 'Limiter' object has no attribute 'ratelimit'
[2025-10-07T10:16:50.789+0000] {logging_mixin.py:190} INFO - ⚠️ Lỗi khi xử lý PTB: 'Limiter' object has no attribute 'ratelimit'
[2025-10-07T10:16:50.791+0000] {logging_mixin.py:190} INFO - ⚠️ Lỗi khi xử lý PSW: 'Limiter' object has no attribute 'ratelimit'
[2025-10-07T10:16:50.811+0000] {logging_mixin.py:190} INFO - ⚠️ Lỗi khi xử lý PSP: 'Limiter' object has no attribute 'ratelimit'
[2025-10-07T10:16:50.819+0000] {logging_mixin.py:190} INFO - ⚠️ Lỗi khi xử lý PSN: 'Limiter' object has no attribute 'ratelimit'
[2025-10-07T10:16:50.821+0000] {logging_mixin.py:190} INFO - ⚠️ Lỗi khi xử lý PSL: 'Limiter' object has no attribute 'ratelimit'
[2025-10-07T10:16:50.897+0000] {logging_mixin.py:190} INFO - ⚠️ Lỗi khi xử lý PSI: 'Limiter' object has no attribute 'ratelimit'
[2025-10-07T10:16:50.899+0000] {logging_mixin.py:190} INFO - ⚠️ Lỗi khi xử lý PSH: 'Limiter' object has no attribute 'ratelimit'
[2025-10-07T10:16:50.900+0000] {logging_mixin.py:190} INFO - ⚠️ Lỗi khi xử lý PSG: 'Limiter' object has no attribute 'ratelimit'
[2025-10-07T10:16:50.901+0000] {logging_mixin.py:190} INFO - ⚠️ Lỗi khi xử lý PSE: 'Limiter' object has no attribute 'ratelimit'
[2025-10-07T10:16:50.902+0000] {logging_mixin.py:190} INFO - ⚠️ Lỗi khi xử lý PSD: 'Limiter' object has no attribute 'ratelimit'
[2025-10-07T10:16:50.902+0000] {logging_mixin.py:190} INFO - ⚠️ Lỗi khi xử lý PSC: 'Limiter' object has no attribute 'ratelimit'
[2025-10-07T10:16:50.904+0000] {logging_mixin.py:190} INFO - ⚠️ Lỗi khi xử lý PSB: 'Limiter' object has no attribute 'ratelimit'
[2025-10-07T10:16:50.905+0000] {logging_mixin.py:190} INFO - ⚠️ Lỗi khi xử lý PRT: 'Limiter' object has no attribute 'ratelimit'
[2025-10-07T10:16:50.906+0000] {logging_mixin.py:190} INFO - ⚠️ Lỗi khi xử lý PRO: 'Limiter' object has no attribute 'ratelimit'
[2025-10-07T10:16:50.921+0000] {logging_mixin.py:190} INFO - ⚠️ Lỗi khi xử lý PRE: 'Limiter' object has no attribute 'ratelimit'
[2025-10-07T10:16:50.924+0000] {logging_mixin.py:190} INFO - ⚠️ Lỗi khi xử lý PRC: 'Limiter' object has no attribute 'ratelimit'
[2025-10-07T10:16:50.925+0000] {logging_mixin.py:190} INFO - ⚠️ Lỗi khi xử lý PQN: 'Limiter' object has no attribute 'ratelimit'
[2025-10-07T10:16:50.928+0000] {logging_mixin.py:190} INFO - ⚠️ Lỗi khi xử lý PPY: 'Limiter' object has no attribute 'ratelimit'
[2025-10-07T10:16:50.931+0000] {logging_mixin.py:190} INFO - ⚠️ Lỗi khi xử lý PPT: 'Limiter' object has no attribute 'ratelimit'
[2025-10-07T10:16:50.937+0000] {python.py:240} INFO - Done. Returned value was: None
[2025-10-07T10:16:51.010+0000] {taskinstance.py:341} INFO - ::group::Post task execution logs
[2025-10-07T10:16:51.019+0000] {taskinstance.py:353} INFO - Marking task as SUCCESS. dag_id=vnstock_weekly_etl_incremental, task_id=process_batch, run_id=manual__2025-10-07T10:01:57.519057+00:00, map_index=12, execution_date=20251007T100157, start_date=20251007T101642, end_date=20251007T101651
[2025-10-07T10:16:51.529+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-10-07T10:16:51.914+0000] {taskinstance.py:3901} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-10-07T10:16:51.943+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
